src/scene/materials/standard-material.js

```diff
@@ -1096,17 +1098,12 @@ function _defineColor(name, defaultValue) {
     defineUniform(name, (material, device, scene) => {
         const uniform = material._allocUniform(name, () => new Float32Array(3));
         const color = material[name];
-        const gamma = material.useGammaTonemap && scene.rendering.gammaCorrection;
 
-        if (gamma) {
-            uniform[0] = Math.pow(color.r, 2.2);
-            uniform[1] = Math.pow(color.g, 2.2);
-            uniform[2] = Math.pow(color.b, 2.2);
-        } else {
-            uniform[0] = color.r;
-            uniform[1] = color.g;
-            uniform[2] = color.b;
-        }
+        // uniforms are always in linear space
+        const uniformColor = _tempColor.linear(color);
```

Comment:
Why do you need a temp var here? Just set `uniform` from `_tempColor` components.

Reply:
yep done, thanks